backend/dal/Repositories/DocumentRepository.cs

```diff
@@ -67,14 +67,8 @@ public PimsDocument Update(PimsDocument document, bool commitTransaction = true)
             document.ThrowIfNull(nameof(document));
 
             this.User.ThrowIfNotAuthorized(Permissions.DocumentEdit);
-            var existingDocument = this.Context.PimsDocuments.Where(l => l.DocumentId == document.DocumentId).FirstOrDefault()
-                 ?? throw new KeyNotFoundException();
-            Context.Entry(existingDocument).CurrentValues.SetValues(document);
-            if (commitTransaction)
-            {
-                this.Context.CommitTransaction();
-            }
-            return existingDocument;
+            document = Context.Update(document).Entity;
```

Comment:
This doesn't allow for related entities to be modified does it?

(one nice thing about context.entry(blah).currentvalues.setValues is that you can only load the relationships that are allowed to be modified whereas Update() will affect anything in the passed object.

Reply:
I don't believe it will. At the moment the document is quite simple and only has the status as a related table